com.unity.ml-agents/Runtime/Actuators/IActionReceiver.cs

```diff
@@ -61,6 +61,36 @@ public ActionBuffers(ActionSegment<float> continuousActions, ActionSegment<int>
             DiscreteActions = discreteActions;
         }
 
+        /// <summary>
+        /// Create an <see cref="ActionBuffers"/> instance with ActionSpec and all actions stored as a float array.
+        /// </summary>
+        /// <param name="actionSpec"><see cref="ActionSpec"/> of the <see cref="ActionBuffers"/></param>
+        /// <param name="actions">The float array of all actions, including discrete and continuous actions.</param>
+        /// <returns>An <see cref="ActionBuffers"/> instance initialized with a <see cref="ActionSpec"/> and a float array.
+        public static ActionBuffers FromActionSpec(ActionSpec actionSpec, float[] actions)
```

Comment:
Let's leave this as `internal` for now, since the interface might change later - we're still not sure how the inference results will be passed back from training, but we might get the int[] values too.